src/main/java/domain/LottoPrizeType.java

```diff
@@ -0,0 +1,34 @@
+package domain;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum LottoPrizeType {
+    FOURTH_PRIZE(3, 5000),
+    THIRD_PRIZE(4, 50000),
+    SECOND_PRIZE(5, 1500000),
+    FIRST_PRIZE(6, 2000000000);
```

Comment:
```suggestion
    FIRST_PRIZE(6, 2_000_000_000);
```

언더바를 통해 가독성을 상승시킬 수 있습니다 :)